src/typebox.ts

```diff
@@ -81,7 +97,7 @@ export type StringFormatOption =
     | 'uuid'         | 'iri-reference' | 'uri-template' | 'json-pointer' | 'relative-json-pointer'
     | 'regex'
 
-export declare type StringOptions<TFormat extends string> = {
+export declare type StringOptions<TFormat extends string | undefined = undefined> = {
```

Comment:
As the `format` property was optional, there's no harm on extending the `TFormat` type constrain, and to give it a default value. This is actually useful when defining the `BrandedString` function.